tests/nested/core/coreconfig-services/task.yaml

```diff
@@ -0,0 +1,22 @@
+summary: Disable and enable back core services via snap set with reboot.
+
+systems: [ubuntu-18.04-64]
+
+execute: |
+  #shellcheck source=tests/lib/nested.sh
+  . "$TESTSLIB/nested.sh"
+  nested_exec "systemctl status systemd-resolved.service" | MATCH "Active: +active"
+
+  echo "Disabling systemd-resolved service"
+  nested_exec "sudo snap set system service.systemd-resolved.disable=true"
+  nested_exec "systemctl status systemd-resolved.service" | MATCH "Active: +inactive"
+  nested_exec "sudo reboot"
```

Comment:
is this reboot right?  without the waiting?

Reply:
ah, nope, copy-paste error, thanks!